Formula/docker-clean.rb

```diff
@@ -0,0 +1,14 @@
+class DockerClean < Formula
+  desc "Script cleans Docker containers, images, networks, and volumes."
+  homepage "https://github.com/ZZROTDesign/docker-clean"
+  url "https://github.com/ZZROTDesign/docker-clean/archive/v2.0.3.tar.gz"
+  sha256 "a2e1923fe2862ca2487e39000c68cba900711b768a8920dd1b1275a2fa0e6789"
+
+  def install
```

Comment:
This formula doesn't need any compilation or other similar things, thus it doesn't make sense to provide a bottle (binary package) for it. Please add `bottle :unneeded` followed by an empty line above this line here.